xdem/examples.py

```diff
@@ -57,8 +62,69 @@ def download_longyearbyen_examples(overwrite: bool = False):
     dir_name = os.path.join(
         temp_dir.name,
         [dirname for dirname in os.listdir(temp_dir.name) if os.path.isdir(os.path.join(temp_dir.name, dirname))][0],
-        "data/Longyearbyen"
+        "data","Longyearbyen"
     )
 
     # Copy the data to the examples directory.
-    copy_tree(dir_name, os.path.join(EXAMPLES_DIRECTORY, "Longyearbyen/data"))
+    copy_tree(dir_name, os.path.join(EXAMPLES_DIRECTORY, "Longyearbyen","data"))
+
+def process_coregistered_examples(overwrite: bool =False):
+    """
+       Process the Longyearbyen example files into a dDEM (to avoid repeating this in many test/documentation steps).
+
+       :param overwrite: Do not download the files again if they already exist.
+       """
+
+    def mkdir_p(out_dir):
```

Comment:
It's not so useful to define a function that is only used once... Also I'm not sure I understand what the function does, it seems to just pass on error...
Would that not solve your problem? https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir

Reply:
I feel old sometimes when I write Python code... but not so long ago the `exist_ok` argument of `pathlib.Path.mkdir` did not exist with `parent=True` and it was a nightmare to deal with those Exceptions regularly! 
So I always had my homemade function to create a parent directoy without any error :)